Runtime/Core/InstalledJavaVersionCheck/InstalledJavaVersionProvider.cs

```diff
@@ -57,10 +57,11 @@ public CheckInstalledJavaVersionResponse CheckInstalledJavaVersion(CheckInstalle
             }
 
             // example output: java version "1.8.0"
-            string[] outputWords = processOutput
+            var versionLine = outputMatch.Groups[0].ToString();
```

Comment:
Will there a NullPointerException if java is not installed? Should we add a null check here?

Reply:
If the [regex match is Success](https://github.com/aws/amazon-gamelift-plugin-unity/pull/69/files#diff-5592f4d287b3fee32c2bb5ad367e9ac2a29a53ffc20cdba78d0ba6ac741e2833R64), there won't be nulls